### DISCUSSION TITLE
BT-23-Lot

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#540] [Q&A] [answered]
Hi Lab
I am using sdk 1.7.
in some notices like 15 this is the definition of BT-23-Lot : 
"id" : "BT-23-Lot",
          "contentType" : "field",
          **"displayType" : "RADIO",**
          "description" : "Main Nature",
          "_label" : "field|name|BT-23-Lot"
        },

while in fields.json : 
"id" : "BT-23-Lot",
    "parentNodeId" : "ND-LotProcurementScope",
    "name" : "Main Nature",
    "btId" : "BT-23",
    "xpathAbsolute" : "/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:ProcurementProject/cbc:ProcurementTypeCode[@listName='contract-nature']",
    "xpathRelative" : "cbc:ProcurementTypeCode[@listName='contract-nature']",
    "xsdSequenceOrder" : [ { "cbc:ProcurementTypeCode" : 5 } ],
    **"type" : "code",
    "legalType" : "CODE",**
    
It seems an error, what do you think ?
Bye
Renato

**Comment by rouschr** [comment, marked as answer]
RADIO on a codelist with more than two values, does look strange.
I will see what can be done.
This should get fixed in SDK 1.9

**Reply by rengit62** [reply]
Till now, I saw this situation only in notice subtype 15 and 16
